cypress/integration/chatFlows/messageOptions.spec.js

```diff
@@ -0,0 +1,74 @@
+describe('Chat message options', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.intercept(
+      { method: 'POST', url: '/chat_channels/1/open' },
+      { body: {} },
+    );
+
+    cy.fixture('users/chatUser1.json').as('user');
+    cy.fixture('users/chatUser2.json').as('user2');
+
+    cy.get('@user').then((user) => {
+      cy.loginUser(user).then(() => {
+        cy.visit('/connect');
+      });
+    });
+  });
+
+  it('should show message option menus', () => {
```

Comment:
I'm going to re-run the frontend tests job a few times in Travis to make sure this is stable - I initially had some issues and it's got a lot of steps, so I'll try to be extra sure I'm not introducing anything potentially flaky

Reply:
Ran 5 times and all green, so I think we're good ✅